src/src/rx_main.cpp

```diff
@@ -1012,51 +1012,89 @@ void UpdateModelMatch(uint8_t model)
  **/
 void MspReceiveComplete()
 {
-    if (MspData[7] == MSP_SET_RX_CONFIG && MspData[8] == MSP_ELRS_MODEL_ID)
-    {
-        UpdateModelMatch(MspData[9]);
-    }
-    else if (MspData[0] == MSP_ELRS_SET_RX_WIFI_MODE)
-    {
+    /* Possible data types received in a MSP frame:
```

Comment:
I don't think that this list is needed here - we can add some comment about the special structure of the VTX or model match package but general information about crsf headers/ext headers seems a bit random here

Reply:
removed comment